infrastructure/lambda-lloyd-george-record-stitch.tf

```diff
@@ -90,6 +87,7 @@ module "lloyd-george-stitch-lambda" {
     LLOYD_GEORGE_DYNAMODB_NAME = "${terraform.workspace}_${var.lloyd_george_dynamodb_table_name}"
     SPLUNK_SQS_QUEUE_URL       = try(module.sqs-splunk-queue[0].sqs_url, null)
     WORKSPACE                  = terraform.workspace
+    PRE_SIGN_ASSUME_ROLE       = aws_iam_role.stitch_pre_sign_url_role.arn
```

Comment:
Is there any way we can change the 'pre_sign'/'PRE_SIGN' strings to 'presigned'/'PRESIGNED' instead to match the proper terminology?